frontend/functions/HelloWorld.ts

```diff
@@ -0,0 +1,3 @@
+export default function HelloWorld() {
```

Comment:
Should you call functions "utils" maybe? The idea is that you will have tests on the utils to make sure all the core util functions work.